crispy_forms/bootstrap.py

```diff
@@ -414,3 +414,57 @@ def __init__(self, field, *args, **kwargs):
 
 class InlineField(Field):
     template = "%s/layout/inline_field.html"
+
+
+class Modal(LayoutObject):
+    """
+    Boostrap layout object for rendering crispy forms objects inside a bootstrap modal.
+
+    The following attributes can be set:
+        - `css_id`: modal's DOM id
+        - `css_class`: modal's DOM classes
+            - NOTE: "modal" and "fade" are applied by default
+        - `title`: text to display in the modal's header
+            - NOTE: text will be wrapped in a <h5> tag
+        - `title_id`: title's DOM id
+        - `title_class`: titles's DOM classes
+            - NOTE: "modal-title" is applied by default
+
+    Example::
+
+        Modal(
+            'field1',
+            Div('field2'),
+            css_id="modal-id-ex",
+            css_class="modal-class-ex,
+            title="This is my modal"
+        )
+    """
+
+    template = "%s/layout/modal.html"
+
+    def __init__(self, *fields, **kwargs):
+        self.fields = list(fields)
+        self.template = kwargs.pop("template", self.template)
+
+        self.css_id = kwargs.pop("css_id", "modal_id")
+        self.title = kwargs.pop("title", "Modal Title")
```

Comment:
Can we test for escaping here? What happens if I add my title as "Tom & Jerry Survey"?

Reply:
Does this still apply even after I changed how kwargs are handled?

Reply:
So if I pass in a title as `This is my & modal` the output is `This is my &amp; modal`. I think that's fine as folk can pass their title in as SafeText if they want to this. As far as I know that's standard practice with the exception of `help_text`.